### PR TITLE
Charlie NTR #203 and #204

### PR DESCRIPTION
**Pull request by cbarcl01**
Added new terms from issues #203 and #204 plus updated existing terms to include appropriate examples of usage or to update legacy definitions.

### CODE REVIEW

File: src/ontology/foodon-edit.owl [hunk 1]
```diff
@@ -13172,6 +13181,9 @@ AnnotationAssertion(obo:IAO_0000114 obo:FOODON_00002465 obo:IAO_0000122)
 
 # Class: obo:FOODON_00002470 (food object quality)
 
+AnnotationAssertion(obo:IAO_0000114 obo:FOODON_00002470 obo:IAO_0000122)
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_00002470 "A physical object quality which can describe an attribute of a food item or related to a food product."@en)
+AnnotationAssertion(obo:IAO_0000117 obo:FOODON_00002470 "0000-0001-8008-8249")
```
Comment (cmrn-rhi) [inline src/ontology/foodon-edit.owl:13186]: When manually adding yourself as term editor you need to include the purl prefix. Protege does this for you automatically for new terms but not for every time you add the annotation.

e.g. use http://orcid.org/0000-0001-8008-8249 like you did in line 66605

```suggestion
AnnotationAssertion(dc:creator obo:FOODON_00002470 <http://orcid.org/0000-0001-8008-8249>)
```

File: src/ontology/foodon-edit.owl [hunk 2]
```diff
@@ -13230,7 +13242,7 @@ AnnotationAssertion(rdfs:label obo:FOODON_00002516 "gravlax"@en)
 
 # Class: obo:FOODON_00002517 (food packing medium compromised)
 
-AnnotationAssertion(obo:IAO_0000112 obo:FOODON_00002517 "An opened can or bottle; a torn plastic cheese wrapper; a breached wax cheese rind."@en)
+AnnotationAssertion(obo:IAO_0000112 obo:FOODON_00002517 "A can of chickpeas with a packing liquid that is colonized with a pathogen."@en)
 AnnotationAssertion(obo:IAO_0000114 obo:FOODON_00002517 obo:IAO_0000122)
 AnnotationAssertion(obo:IAO_0000115 obo:FOODON_00002517 "A compromised food packing medium is a gaseous, liquid, or solid medium that has been exposed to the environment due to a failure in integrity of the food packaging.  The packaging failure may be intentional, e.g. opening a bag, or unintentional, e.g. a tear."@en)
 AnnotationAssertion(obo:IAO_0000117 obo:FOODON_00002517 "Damion Dooley"^^xsd:string)
```
Comment (cmrn-rhi) [inline src/ontology/foodon-edit.owl:13248]: Add a `term editor` annotation for yourself in addition to Damion's. 

```suggestion
AnnotationAssertion(obo:IAO_0000117 obo:FOODON_00002517 "Damion Dooley"^^xsd:string)
AnnotationAssertion(obo:IAO_0000117 obo:FOODON_00002517 <http://orcid.org/0000-0001-8008-8249>)
```

File: src/ontology/foodon-edit.owl (hunk 1 above)
Comment (cmrn-rhi) [reply src/ontology/foodon-edit.owl:13186]: I've noticed that outside of manual curation, we primarily use `term editor` [IAO_0000117] (e.g., in ROBOT tables) even for new terms. Going to check with Damion if he'd like us to only use `term editor` moving forward. If so we'll have to disable Protege's automated `dc:creator` annotation upon creation of new terms.

File: src/ontology/foodon-edit.owl [hunk 3]
```diff
@@ -65857,17 +65872,22 @@ SubClassOf(obo:FOODON_03530217 obo:FOODON_03530008)
 
 # Class: obo:FOODON_03530218 (food packing medium integrity)
 
-AnnotationAssertion(obo:IAO_0000114 obo:FOODON_03530218 obo:IAO_0000428)
-AnnotationAssertion(obo:IAO_0000115 obo:FOODON_03530218 "It is important to distinguish whether the food in question has been drained or not drained when evaluating the nutrient content.
-Related term to PRESENCE OF PACKING MEDIUM is K.  PACKING MEDIUM."@en)
+AnnotationAssertion(obo:IAO_0000112 obo:FOODON_03530218 "A can of peaches in syrup, in which the syrup is uncompromised."@en)
+AnnotationAssertion(obo:IAO_0000114 obo:FOODON_03530218 obo:IAO_0000122)
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_03530218 "A food object quality which describes the integrity of the medium in which the food is packed for preservation and handling or the medium surrounding foods."@en)
+AnnotationAssertion(obo:IAO_0000116 obo:FOODON_03530218 "Existing definition was a legacy definition, not ontological and has been moved to comments. Updated examples of usage to remove plastic usage as describes packaging not packing medium."@en)
```
Comment (cmrn-rhi) [inline src/ontology/foodon-edit.owl:65878]: When mentioning `existing` you are now referencing the current definition, not the retired one, as the context of it's application changed when the definition did. Recommend using "Previous" like you did in the heading of your comment.

```suggestion
AnnotationAssertion(obo:IAO_0000116 obo:FOODON_03530218 "Previous definition was a legacy definition, not ontological and has been moved to comments. Updated examples of usage to remove plastic usage as describes packaging not packing medium."@en)
```

File: src/ontology/foodon-edit.owl [hunk 4]
```diff
@@ -58457,8 +58469,10 @@ SubClassOf(obo:FOODON_03480047 obo:FOODON_03480029)
 
 # Class: obo:FOODON_03490100 (food packaging)
 
-AnnotationAssertion(obo:IAO_0000114 obo:FOODON_03490100 obo:IAO_0000428)
-AnnotationAssertion(obo:IAO_0000115 obo:FOODON_03490100 "Type of container or wrapping defined by the main container material, the container form, and the material of the liner lids or ends. Also type of container or wrapping by form; prefer description by material first, then by form.  [FDA CFSAN 1995]"@en)
+AnnotationAssertion(obo:IAO_0000114 obo:FOODON_03490100 obo:IAO_0000122)
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_03490100 "A food contact material which encloses the food and can be in the type of a container, packaging or wrapping."@en)
+AnnotationAssertion(obo:IAO_0000116 obo:FOODON_03490100 "Existing definition was a legacy definition, not ontological and has been moved to comments."@en)
```
Comment (cmrn-rhi) [inline src/ontology/foodon-edit.owl:58474]: When mentioning `existing` you are now referencing the current definition, not the retired one, as the context of it's application changed when the definition did. Recommend using "Previous" like you did in the heading of your comment.
```suggestion
AnnotationAssertion(obo:IAO_0000116 obo:FOODON_03490100 "Previous definition was a legacy definition, not ontological and has been moved to comments."@en)
```

File: src/ontology/foodon-edit.owl [hunk 5]
```diff
@@ -58457,8 +58469,10 @@ SubClassOf(obo:FOODON_03480047 obo:FOODON_03480029)
...
+AnnotationAssertion(obo:IAO_0000119 obo:FOODON_03490100 "https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/food-packaging")
 AnnotationAssertion(obo:IAO_0000412 obo:FOODON_03490100 "http://langual.org")
```
Comment (cmrn-rhi) [inline src/ontology/foodon-edit.owl:58476]: Add yourself to the edit history with a `term editor` annotation (I picked this line arbitrarily).

```suggestion
AnnotationAssertion(obo:IAO_0000412 obo:FOODON_03490100 "http://langual.org")
AnnotationAssertion(obo:IAO_0000117 obo:FOODON_03490100 <http://orcid.org/0000-0001-8008-8249>)
```

File: src/ontology/foodon-edit.owl [hunk 6]
```diff
@@ -65857,17 +65872,22 @@ SubClassOf(obo:FOODON_03530217 obo:FOODON_03530008)
...
 # Class: obo:FOODON_03530218 (food packing medium integrity)
 
-AnnotationAssertion(obo:IAO_0000114 obo:FOODON_03530218 obo:IAO_0000428)
-AnnotationAssertion(obo:IAO_0000115 obo:FOODON_03530218 "It is important to distinguish whether the food in question has been drained or not drained when evaluating the nutrient content.
-Related term to PRESENCE OF PACKING MEDIUM is K.  PACKING MEDIUM."@en)
+AnnotationAssertion(obo:IAO_0000112 obo:FOODON_03530218 "A can of peaches in syrup, in which the syrup is uncompromised."@en)
+AnnotationAssertion(obo:IAO_0000114 obo:FOODON_03530218 obo:IAO_0000122)
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_03530218 "A food object quality which describes the integrity of the medium in which the food is packed for preservation and handling or the medium surrounding foods."@en)
+AnnotationAssertion(obo:IAO_0000116 obo:FOODON_03530218 "Existing definition was a legacy definition, not ontological and has been moved to comments. Updated examples of usage to remove plastic usage as describes packaging not packing medium."@en)
 AnnotationAssertion(obo:IAO_0000412 obo:FOODON_03530218 "http://langual.org")
```
Comment (cmrn-rhi) [inline src/ontology/foodon-edit.owl:65879]: Add yourself to the edit history with a `term editor` annotation (I picked this line arbitrarily).

```suggestion
AnnotationAssertion(obo:IAO_0000412 obo:FOODON_03530218 "http://langual.org")
AnnotationAssertion(obo:IAO_0000117 obo:FOODON_03530218 <http://orcid.org/0000-0001-8008-8249>)
```

File: src/ontology/foodon-edit.owl [hunk 7]
```diff
@@ -65857,17 +65872,22 @@ SubClassOf(obo:FOODON_03530217 obo:FOODON_03530008)
...
+Related term to PRESENCE OF PACKING MEDIUM is K.  PACKING MEDIUM."@en)
 AnnotationAssertion(rdfs:label obo:FOODON_03530218 "food packing medium integrity"@en)
 SubClassOf(obo:FOODON_03530218 obo:FOODON_00002470)
 
 # Class: obo:FOODON_03530219 (food packing medium drained)
 
+AnnotationAssertion(obo:IAO_0000112 obo:FOODON_03530219 "A can of chickpeas where the liquid medium has been drained."@en)
 AnnotationAssertion(obo:IAO_0000114 obo:FOODON_03530219 obo:IAO_0000428)
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_03530219 "A food object quality where the medium in which the food is packed is drained."@en)
 AnnotationAssertion(obo:IAO_0000412 obo:FOODON_03530219 "http://langual.org")
```
Comment (cmrn-rhi) [inline src/ontology/foodon-edit.owl:65891]: Add yourself to the edit history with a term editor annotation (I picked this line arbitrarily).

```suggestion
AnnotationAssertion(obo:IAO_0000412 obo:FOODON_03530219 "http://langual.org")
AnnotationAssertion(obo:IAO_0000117 obo:FOODON_03530219 <http://orcid.org/0000-0001-8008-8249>)
```

File: src/ontology/foodon-edit.owl [hunk 8]
```diff
@@ -65876,7 +65896,10 @@ SubClassOf(obo:FOODON_03530219 obo:FOODON_03530218)
 
 # Class: obo:FOODON_03530220 (food packing medium not drained)
 
+AnnotationAssertion(obo:IAO_0000112 obo:FOODON_03530220 "A can of chickpeas where the liquid medium has not been drained."@en)
 AnnotationAssertion(obo:IAO_0000114 obo:FOODON_03530220 obo:IAO_0000428)
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_03530220 "A food object quality where the medium in which the food is packed is still surrounding the food."@en)
+AnnotationAssertion(obo:IAO_0000117 obo:FOODON_03530220 "0000-0001-8008-8249"@en)
```
Comment (cmrn-rhi) [inline src/ontology/foodon-edit.owl:65902]: Add ORCiD PURL prefix and don't include an `@en` tag (probably an accidental carry over form a previous annotation).

```suggestion
AnnotationAssertion(obo:IAO_0000117 obo:FOODON_03530220 <http://orcid.org/0000-0001-8008-8249>)
```

File: src/ontology/foodon-edit.owl [hunk 9]
```diff
@@ -66564,6 +66587,93 @@ AnnotationAssertion(oboInOwl:hasDbXref obo:FOODON_03530295 "http://www.langual.o
 AnnotationAssertion(rdfs:label obo:FOODON_03530295 "foot (poultry meat cut)"@en)
 SubClassOf(obo:FOODON_03530295 obo:FOODON_03530156)
 
+# Class: obo:FOODON_03601000 (food packaging integrity)
+
+AnnotationAssertion(obo:IAO_0000112 obo:FOODON_03601000 "A compromised plastic food wrapper."@en)
+AnnotationAssertion(obo:IAO_0000114 obo:FOODON_03601000 obo:IAO_0000122)
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_03601000 "A food object quality which describes the stability and completeness of the packaging material."@en)
+AnnotationAssertion(obo:IAO_0000117 obo:FOODON_03601000 "0000-0001-8008-8249")
```
Comment (cmrn-rhi) [inline src/ontology/foodon-edit.owl:66595]: Add ORCiD PURL prefix.

```suggestion
AnnotationAssertion(obo:IAO_0000117 obo:FOODON_03601000 <http://orcid.org/0000-0001-8008-8249>)
```

**Review by cmrn-rhi — CHANGES_REQUESTED** [review]
Have recommended some small edits to do before passing it off to Damion.
Also waiting to hear back from Damion on exact preferences for `term editor` and `dc:creator` usage as that may have (minor) impact on a few annotations.

**Comment by cbarcl01** [comment]
Changes made in Protege under a new commit and a new pull request generated.

File: src/ontology/foodon-edit.owl (hunk 1 above)
Comment (ddooley) [reply src/ontology/foodon-edit.owl:13186]: I think protege preferences -> "new entities metadata" allows you to control exactly which tags it adds.  I'm thinking like nico we should switch to dc:contributor !!

Comment (cmrn-rhi) [reply src/ontology/foodon-edit.owl:13186]: Right, forgot we could change it - so let's use `dc:contributor` as default and can discuss this further in Oc/Or to make sure everyone is on board.
image

File: src/ontology/foodon-edit.owl (hunk 2 above)
Comment (cmrn-rhi) [reply src/ontology/foodon-edit.owl:13248]: Resolved in later commits.

File: src/ontology/foodon-edit.owl (hunk 4 above)
Comment (cmrn-rhi) [reply src/ontology/foodon-edit.owl:58474]: Resolved in later commits.

File: src/ontology/foodon-edit.owl (hunk 6 above)
Comment (cmrn-rhi) [reply src/ontology/foodon-edit.owl:65879]: Resolved in later commits.

File: src/ontology/foodon-edit.owl (hunk 7 above)
Comment (cmrn-rhi) [reply src/ontology/foodon-edit.owl:65891]: Resolved in later commits.

File: src/ontology/foodon-edit.owl (hunk 5 above)
Comment (cmrn-rhi) [reply src/ontology/foodon-edit.owl:58476]: Resolved in later commits.

File: src/ontology/foodon-edit.owl (hunk 8 above)
Comment (cmrn-rhi) [reply src/ontology/foodon-edit.owl:65902]: Resolved in later commits.

File: src/ontology/foodon-edit.owl (hunk 9 above)
Comment (cmrn-rhi) [reply src/ontology/foodon-edit.owl:66595]: Resolved in later commits.

File: src/ontology/foodon-edit.owl (hunk 3 above)
Comment (cmrn-rhi) [reply src/ontology/foodon-edit.owl:65878]: Resolved in later commits.

**Pull request closed without merging**